apps/mobile/package.json

```diff
@@ -79,6 +79,8 @@
     "@stacks/transactions": "7.0.5",
     "@stacks/wallet-sdk": "7.0.2",
     "@tanstack/react-query": "5.59.16",
+    "@types/lodash.get": "4.4.9",
```

Comment:
Should we add an explicit `react` version resolution to `mobile` to stop us hitting this problem again? 

It could at least serve as a reminder of the fixed version we need until we manage to upgrade React